Explorer/Assets/DCL/PerformanceAndDiagnostics/Diagnostics/ReportsHandling/Handlers/DebugLogReportHandler.cs

```diff
@@ -111,10 +112,19 @@ private static string GetReportDataPrefix(in ReportData reportData)
         {
             string color = GetCategoryColor(in reportData);
 
+            var debugLogBuilder = new StringBuilder();
+            debugLogBuilder.Append($"<color=#{color}>");
+            debugLogBuilder.Append($"[{reportData.Category}]");
+
             if (reportData.SceneShortInfo.BaseParcel != Vector2Int.zero)
-                return $"<color=#{color}>[{reportData.Category}] {reportData.SceneShortInfo.BaseParcel}</color>: ";
+                debugLogBuilder.Append($" {reportData.SceneShortInfo.BaseParcel}");
+
+            if (reportData.SceneTickNumber != null)
+                debugLogBuilder.Append($" [T: {reportData.SceneTickNumber}]");
+
+            debugLogBuilder.Append("</color>: ");
 
-            return $"<color=#{color}>[{reportData.Category}]</color>: ";
+            return debugLogBuilder.ToString();
```

Comment:
Would it be faster? String interpolation worked as SB by a single bundle of strings

Reply:
I changed it because I introduced another optional parameter, otherwise I would need to go with a third "if"